lib/invest/invest-service.ts

```diff
@@ -40,23 +46,66 @@ export namespace investService {
     return playerInfos;
   }
 
+  export async function reListLowPlayer(): Promise<string[]> {
```

Comment:
I think this is more of a trade-bot job than invest job. Let's move it there

Reply:
Check 👍